typedb-src/modules/ROOT/pages/admin/configuration.adoc

```diff
@@ -157,7 +157,14 @@ The log section of the configuration file contains the logging behavior options
 
 There are three subsections:
 
-* `output`: Define destinations to write logs to. Allowed types are `type: file`, and `type: stdout` in TypeDB.
+* `output`: Define destinations to write logs to.
+** User-defined output channel name
+*** `type` -- it's either `file` or `stdout`.
+*** `directory` -- filepath, relative to the server binary. Only available for `type: file`.
+*** `file-size-cap` -- maximum size of a log file. If the log file reaches the limit, we start writing in a new file
```

Comment:
I think in these "option documentation" type descriptions we have tended to stay away from "we" (since it's from the perspective of TypeDB's operation anyway). I think that is still being held to? In this case, we can reduce to `If the log file reaches the limit, start writing in a new file` or something like that?

Reply:
> I think in these "option documentation" type descriptions we have tended to stay away from "we" (since it's from the perspective of TypeDB's operation anyway). I think that is still being held to? In this case, we can reduce to `If the log file reaches the limit, start writing in a new file` or something like that?

Our style guide allows we. And I think the suggested variant doesn't look right. It seems like a user's instruction to start writing a new file somehow. I'll try to find a way to rewrite this to drop the pronoun tomorrow.

Reply:
Addressed the issue in https://github.com/vaticle/typedb-docs/pull/688/commits/3acbc02d2bb430adf1dc48b2c616a0d8d773c87f